### PR TITLE
OCPBUGS#38075: Adding a note about isRdma

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.12+

Issue:
[OCPBUGS-38075](https://issues.redhat.com/browse/OCPBUGS-38075)

Link to docs preview:
- [SR-IOV network node configuration object](https://84790--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-sriov-device.html)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@SchSeba 
Could you please review this PR?
Thanks!

**Review by SchSeba — APPROVED** [review]
/lgtm

**Comment by sr1kar99** [comment]
@evgenLevin 
Could you please review this PR?
Thanks!

File: modules/nw-sriov-networknodepolicy-object.adoc [hunk 1]
```diff
@@ -99,6 +99,11 @@ For a Mellanox NIC to work in DPDK mode on bare metal nodes, use the `netdevice`
 If the `isRdma` parameter is set to `true`, you can continue to use the RDMA-enabled VF as a normal network device. A device can be used in either mode.
 +
 Set `isRdma` to `true` and additionally set `needVhostNet` to `true` to configure a Mellanox NIC for use with Fast Datapath DPDK applications.
++
+[NOTE]
+====
+You cannot set `isRdma` to `true` for intel NICs.
```
Comment (lahinson) [inline modules/nw-sriov-networknodepolicy-object.adoc:105]:
```suggestion
You cannot set the `isRdma` parameter to `true` for intel NICs.
```
Including a noun after `isRdma` for clarity.

**Review by lahinson — COMMENTED** [review]
Just one nit. Otherwise, looks good!

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**